helm/values.yaml

```diff
@@ -7,6 +7,7 @@ app:
   secretHash: secrets-hash
   memoryLimit: "1Gi"
   spring:
+    additionalProfiles: "default-value"
```

Comment:
We'll see how the Publisher team feels about this. Personally I can live with it, but it would be slightly nicer if there were a template function we could use to automatically add/remove the initial comma we need so we can default to an empty string.

Reply:
Oh, I think there is an option. If I'm not mistaken I can use if statements to conditionally place the comma if the variable is defined.